### PR TITLE
Add License CC-BY 4.0 Import exemption

### PR DESCRIPTION
**Pull request by pinkforest**
Allows importing advisories with attribution from GHSA

EDIT: This is so that it will be ready for the tooling https://github.com/rustsec/rustsec/pull/682

### CODE REVIEW

**Comment by Shnatsel** [comment]
I think we will need a more clear way to distinguish what data is under what license. For example a `license` field, or an explicitly specified attribution link since CC-BY requires attribution.

**Comment by pinkforest** [comment]
Yes we cna do it after we have the license but that will require tooling changes.

In the meantime we can simply do a source attribution that seems to be the way nvd does it.

**Comment by Shnatsel** [comment]
That is, I agree with the basic direction, but the implementation will be more complex than this.

FWIW I've already tried talking to Github about a licensing exception for RustSec and they said they aren't going to do it.

**Comment by pinkforest** [comment]
Yeah I get the tooling is complex but having the license exemption approved & ready is a good start I would think ?

**Comment by Shnatsel** [comment]
What I think could be valuable is a note that we're looking into adding advisories under CC-BY in the future, to ensure we can import from GHSA.

**Comment by pinkforest** [comment]
But is that a license exemption issue that needs to be in license.txt ? This is just a legal foundational exemption blurb.

I've noted that importing which can generally by in CC-BY 4.0 term that is defined in the license.txt now

**Comment by pinkforest** [comment]
For anyone ICYMI the tooling change PR is here: https://github.com/rustsec/rustsec/pull/682

**Review by tarcieri — APPROVED** [review]
This seems ok as a start, although it would be good to have a proper attribution for imported advisories with structured data about the license

**Review by Shnatsel — CHANGES_REQUESTED** [review]
The data up until now is exclusively public domain, and I think it's important to capture that.

Please rephrase this to state that CC-BY (or other) content may be added in the future, and that it only applies to licenses with attribution requirements but no additional clauses like copyleft.

**Review by Shnatsel — APPROVED** [review]

File: LICENSE.txt [hunk 1]
```diff
@@ -9,3 +9,12 @@ the extent allowed by law.
 
 You can copy, modify, distribute, and retransmit any information in this
 repository, even for commercial purposes, without asking permission.
+
+Additional adapted content from GitHub Security Advisory ("GHSA") database
+
+CC-BY (or other) additional content may be adapted from GHSA that only applies
+to licenses with attribution requirements but no additional clauses like
+copyleft.
```
Comment (alex) [inline LICENSE.txt:17]: I'm having a bit of trouble parsing this sentence. I think there might be a missing or extra word?

Comment (pinkforest) [reply LICENSE.txt]: How about now?

**Review by alex — APPROVED** [review]

**Review by amousset — APPROVED** [review]

**Comment by amousset** [comment]
This is a good first step, the next one is probably to reach a consensus on how to store this information in a structured way (in https://github.com/rustsec/rustsec/pull/682).
